Yoast/wordpress-seo
# Add styling for image upload component

## Context


* Changes to the upload component so that it uses the monorepo.

## Summary


This PR can be summarized in the following changelog entry:

* Changes to the upload component so that it uses the monorepo.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Pull "14292-image-upload-styling" from the "wordpress-seo" and the "javascript" repository.
* Link the monorepo.
* Head to the "Facebook" tab under the "Social" link.
* Check if the upload components look good.

## UI changes

## Documentation

## Quality assurance


Fixes #14292
